### PR TITLE
Use existing ACLK event loop for cloud queries

### PR DESCRIPTION
**Pull request by stelfrag**
##### Summary
- Refrain from pre-allocating a fixed number of threads for cloud queries.
- Schedule all queries through the ACLK sync event loop (taking advantage of the UV worker pool)
- Assign workers to handle queries in parallel (configure via `aclk query threads` in the `[cloud]` section)
  - If all workers are busy queries are executed synchronously
  - Monitor stats under `netdata monitoring`  `aclksync.jobs_started_by_type`
image

### CODE REVIEW

File: src/database/sqlite/sqlite_aclk.c [hunk 1]
```diff
@@ -338,6 +383,34 @@ static void aclk_synchronization(void *arg)
                 case ACLK_DATABASE_PUSH_ALERT:
                     aclk_push_alert_events_for_all_hosts();
                     break;
+
+                case ACLK_MQTT_WSS_CLIENT:
+                    config->client = (mqtt_wss_client) cmd.param[0];
+                    break;
+
+                case ACLK_QUERY_EXECUTE:;
```
Comment (underhood) [inline src/database/sqlite/sqlite_aclk.c:391]: :;?

Comment (stelfrag) [reply src/database/sqlite/sqlite_aclk.c:391]: just a black statement to bypass an issue in the IDE that would complain if the first statement is a variable definition. I will check if it has been fixed and removed -- it is used in other parts of the code as well.

Comment (stelfrag) [reply src/database/sqlite/sqlite_aclk.c:391]: It gives a warning `Label followed by a declaration is a C23 extension` so the `;` handles that

**Review by thiagoftsm — APPROVED** [review]
PR is working as expected:

Screenshot_20240824_002025

**Pull request merged**